draft-ietf-sframe-enc.md

```diff
@@ -149,60 +147,72 @@ media in a broad range of scenarios, as outlined by the following goals:
    scenarios.
 
 
-#SFrame
-We propose a frame level encryption mechanism that provides effective end-to-end encryption, is simple to implement, has no dependencies on RTP, and minimizes encryption bandwidth overhead. Because SFrame encrypts the full frame, rather than individual packets, bandwidth overhead is reduced by having a single IV and authentication tag for each media frame.
+# SFrame
+
+This document defines an encryption mechanism that provides effective end-to-end encryption, is simple to implement, has no dependencies on RTP, and minimizes encryption bandwidth overhead. Because SFrame can encrypt a full frame, rather than individual packets, bandwidth overhead can reduced by adding encryption overhead only once per media frame, instead of once per packet.
+
+## Application Context
+
+SFrame is a general encryption framing that can be applied before or after
+packetization within a media stack.  That is, SFrame can be applied per-frame or
```

Comment:
I'd prefer we live packetization out of the this document. SFrame may be used in other places where there is no concept of packetization.